assets/style.css

```diff
@@ -1,5 +1,30 @@
+header {
+    background-color:  hsl(0, 0%, 90%);
+    color:  hsl(0, 2%, 45%);
+    margin-bottom: 1.4rem;
```

Comment:
Acredito algo como a classe `mb-1` resolve isso

https://getbootstrap.com/docs/5.2/utilities/spacing/#margin-and-padding

Reply:
Feito